internal/hmyapi/blockchain.go

```diff
@@ -63,6 +64,63 @@ func (s *PublicBlockChainAPI) GetBlockByHash(ctx context.Context, blockHash comm
 	return nil, err
 }
 
+// GetShardingStructure returns an array of sharding structures.
+func (s *PublicBlockChainAPI) GetShardingStructure(ctx context.Context) ([]map[string]interface{}, error) {
+	if core.ShardingSchedule.GetNetworkID() == shardingconfig.MainNet {
```

Comment:
better wrap this data into the schedule, don't hardcode it here.

Reply:
ShardingSchedule is this schedule variable.